generator/parser/codemodel.h

```diff
@@ -512,6 +517,9 @@ class _MemberModelItem: public _CodeModelItem
       uint _M_isExtern: 1;
       uint _M_isMutable: 1;
     };
+#if defined(__GNUC__)
+#pragma GCC diagnostic pop
+#endif
     uint _M_flags;
   };
```

Comment:
Wouldn't it be better to just use an allowed construct, e.g. something like
```cpp
union
{
    struct bitflags_type
    {
        uint _M_isConstant: 1;
        uint _M_isExtern: 1;
        uint _M_isMutable: 1;
    } bitflags;
    uint _M_flags;
};
```

Reply:
Changed to give the structs names rather than disabling the warnings.